### ISSUE TITLE
Please uses Qt5 style aware scrollbars and sliders

### ISSUE THREAD

**Issue by karolherbst** [qTox/qTox#698]
It doesn't matter which Qt5 style I am using, the sliders and scrollbars are always looking the same and not how the one from the style. Also with the breeze Style (used in the upcomming KDE plasma5) the scrollbar has a black background in the settings

**Comment by tux3** [comment]
Can you post a screenshot of what you're seeing and what styles are available in the settings ?

**Comment by karolherbst** [comment]
![qtox_issue698_1](https://cloud.githubusercontent.com/assets/171766/4985564/ba990b1e-692e-11e4-9dfa-59d887b8809f.png)

this one shows the slider in tox on the left side and how the breeze theme should look like in systemsettings

![qtox_issue698_2](https://cloud.githubusercontent.com/assets/171766/4985558/b1fbf818-692e-11e4-9281-9b3b689d7f69.png)

this picture shouws all available themes and the scrollbar

**Comment by karolherbst** [comment]
also changing the theme doesn't change the scrollbar or the slides, except that the black background in the scrollbar is only there with the Breeze theme

**Comment by dubslow** [comment]
@agilob ?

**Comment by agilob** [comment]
Good moment to try KDE5 o/

I think removing some generic definitions from CSSs will fix it, like `QSlider` from mainContent.css

**Comment by karolherbst** [comment]
thanks for the fix so far, but sadly the black background issue isn't fixed with the breeze theme, but the sliders are looking as expected with every style now.

**Comment by karolherbst** [comment]
I hope it isn't a bug inside breeze itself, but others applications are just looking fine. Also the scrollbars doesn't look theme specific either.

**Comment by karolherbst** [comment]
![qtox_issue698_3](https://cloud.githubusercontent.com/assets/171766/5014308/fc495cf0-6a94-11e4-9848-778152f03f2c.png)

I just removed all scrollbar related stuff from the css file and then it looks like that. Maybe it is an background issue somewhere.

**Comment by karolherbst** [comment]
``` diff
diff --git a/ui/settings/mainContent.css b/ui/settings/mainContent.css
index 1b9d4bf..15c7377 100644
--- a/ui/settings/mainContent.css
+++ b/ui/settings/mainContent.css
@@ -77,7 +77,6 @@ QTabBar
 QScrollArea
 {
     background-color: white;
-    background: transparent; 
 }

 QScrollArea > QWidget > QWidget
```

this patch fixes teh black background problem for me, though I have still the custom tox styled scrollbars then. What is the reason behind a custom scrollbar style?

**Comment by agilob** [comment]
> this patch fixes teh black background problem for me, though I have still the custom tox styled scrollbars then. What is the reason behind a custom scrollbar style? 

I don't know, most of the .css files were here before I started contributing. I even never did CSS in Qt before :P

**Comment by karolherbst** [comment]
I see, but personally I think that such styling should be done by the Qt5 theme mainly and not in the application except there is a really good reason why, but I don't see any in this case ;) I think I will test how qtox does look like without any custom declerations.

**Comment by karolherbst** [comment]
Maybe the reason was to have the same style across different tox clients :/ But this is how it looks without any ui/settings/*.css declerations 
![qtox_issue698_4](https://cloud.githubusercontent.com/assets/171766/5015564/26d326ae-6a9f-11e4-8a80-8d591738e384.png)

normally the window background should be also grey for me like in my first screenshot in the systemsettings application :/ Maybe there could be some kind of option like "use tox style" or "use system style"

**Comment by agilob** [comment]
And here, how it looks like with no declarations in `ui/settings/*.css when you have dark system theme:
![virus3](https://cloud.githubusercontent.com/assets/2544251/5016093/133b8748-6a9b-11e4-9354-0a5d6af7414c.png)

Even skype looks better :<

**Comment by karolherbst** [comment]
yeah, this white background isn't right, this comes from somewhere else, but I didn't find it.

**Comment by agilob** [comment]
Do you want and option to completely disable custom css? I know where to do it, it's in designer not, it requires some changes in XML not cpp code.

**Comment by karolherbst** [comment]
yeah would be nice to have such an option, so that the user can decide whether he wants a custom tox desing or its system design.

Though I think, that my scrollbar background is black issues comes from the costom style somehow, because with the patch from https://github.com/tux3/qTox/issues/698#issuecomment-62756397 the issues disappears, but I don't know what else is different with it.
